### PR TITLE
Setup base for server REST api

### PR DESCRIPTION
**Pull request by jclariviere**

### CODE REVIEW

File: common/src/main/java/com/arcbees/beeshop/common/api/ApiPaths.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.arcbees.beeshop.common.api;
+
+public class ApiPaths {
```
Comment (christiangoudreau) [inline common/src/main/java/com/arcbees/beeshop/common/api/ApiPaths.java:19]: common? You have a maven module for "shared"? (it usually sit in the server module)

File: common/src/main/java/com/arcbees/beeshop/common/api/ApiPaths.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    public static final String ROOT = "/api";
+
+    public static final String PAYMENT = "/payment";
+    public static final String STRIPE = "/stripe";
```
Comment (Elrhino) [inline common/src/main/java/com/arcbees/beeshop/common/api/ApiPaths.java:23]: Whats the difference between the payment and stripe paths?

File: common/src/main/java/com/arcbees/beeshop/common/api/ApiPaths.java (hunk 1 above)
Comment (jclariviere) [inline common/src/main/java/com/arcbees/beeshop/common/api/ApiPaths.java:19]: @Chris-V

File: common/src/main/java/com/arcbees/beeshop/common/api/ApiPaths.java (hunk 2 above)
Comment (jclariviere) [inline common/src/main/java/com/arcbees/beeshop/common/api/ApiPaths.java:23]: I extracted a commit from another branch to make this PR, so none for now :) Stripe will be a subpath of payment ("/payment/stripe").

File: common/src/main/java/com/arcbees/beeshop/common/api/ApiPaths.java (hunk 1 above)
Comment (Chris-V) [inline common/src/main/java/com/arcbees/beeshop/common/api/ApiPaths.java:19]: Ça évite de polluer l'auto-complete dans client avec les classes dans .server

Comment (Chris-V) [inline common/src/main/java/com/arcbees/beeshop/common/api/ApiPaths.java:19]: Mais ouais, le package devrait être .shared autrement

File: pom.xml [hunk 3]
```diff
@@ -24,6 +24,7 @@
 
         <javax.inject.version>1</javax.inject.version>
         <servlet-api.version>2.5</servlet-api.version>
+        <resteasy.version>3.0.11.Final</resteasy.version>
```
Comment (meriouma) [inline pom.xml]: 3.0.13

**Comment by meriouma** [comment]
LGTM

**Pull request merged**